lib/ClangImporter/SwiftLookupTable.cpp

```diff
@@ -1945,6 +1946,15 @@ void importer::addEntryToLookupTable(SwiftLookupTable &table,
                          named, importedName.getEffectiveContext());
         }
 
+        if (auto swiftNameAttr = named->getAttr<clang::SwiftNameAttr>()) {
```

Comment:
I think this might need a comment why are we doing this since most people would expect `importedName` to already respect these attributes. But we now started to ignore some of them right? Is that the reason? If that is the case could we run into any trouble if we didn't ignore it and we try to add duplicate entries to the lookup table?